platforms/reference/include/SimTKOpenMMRealType.h

```diff
@@ -75,6 +53,15 @@
 #endif
 #endif
 
+#define PI_M               M_PI
+#define RADIAN             (180/M_PI)
+#define SQRT_TWO           1.41421356237309504
+
+#define DOT3(u,v) ((u[0])*(v[0]) + (u[1])*(v[1]) + (u[2])*(v[2]))
+
+
+// Physical constants are from https://pml.nist.gov/cuu/Constants
```

Comment:
@peastman : This is insufficient provenance information, since it doesn't provide a versioned reference for the physical constants you are using. 

You'll recall from https://github.com/openmm/openmm/issues/2427 that the appropriate standards body is CODATA, and the page that you point to recommends the [CODATA 2018](https://pml.nist.gov/cuu/Constants/) constants.

Let's 
* explicitly state in the header that this is the CODATA 2018 set
* add to the changelog that we have updated to CODATA 2018
* note in the user and dev guides that we use CODATA 2018 physical constants

Reply:
> You'll recall from #2427 that the appropriate standards body is CODATA, and the page that you point to recommends the CODATA 2018 constants.

Look at the URL you just linked to.  Now look at the one in the comment.  Hint: they're the same. :)

Reply:
How does that address any of the three stated actions bullet points above?

That URL hosts pointers to both the latest and historical CODATA sets. The actions I articulate above state that we should *specify which of the sets listed in the URL* we are using. 

We have no control over that site, and it is clear that the newer CODATA version will be posted there when it is available, so simply linking to that site without specifying which version of CODATA there we use is completely unhelpful.